packages/core/contracts/financial-templates/expiring-multiparty/Liquidatable.sol

```diff
@@ -504,6 +504,28 @@ contract Liquidatable is PricelessPositionManager {
         return liquidations[sponsor];
     }
 
+    /**
+     * @notice Accessor method to calculate a transformed collateral requirement using the finanical product library
+      specified during contract deployment. If no library was provided then no modification to the collateral requirement is done.
+     * @param price input price used as an input to transform the collateral requirement.
+     * @return transformedCollateralRequirement collateral requirement with transformation applied to it.
+     * @dev This method should never revert.
+     */
+    function transformCollateralRequirement(FixedPoint.Unsigned memory price)
```

Comment:
Are you purposefully making this public?

Reply:
yes. it's a view function so no risk and it is needed for frontends to compute the current collateral requirement based on a given input price.